system-test/vision.ts

```diff
@@ -79,7 +79,7 @@ describe('Vision', () => {
           responses: [prototypes.google.cloud.vision.v1.IAnnotateImageResponse]
         ) => {
           const response = responses[0];
-          assert.strictEqual(
+          assert.ok(
             response.logoAnnotations![0].description!.toLowerCase(),
```

Comment:
According to the output from #821:

```
Cannot read property 'description' of undefined
TypeError: Cannot read property 'description' of undefined
    at /tmpfs/src/github/nodejs-vision/build/system-test/vision.js:57:60
        -> /tmpfs/src/github/nodejs-vision/system-test/vision.ts:83:42
```

The problem is that `logoAnnotations[0]` is undefined. So, `logoAnnotations` comes back as an array, but there is no object at the 0 index. I wonder why the test doesn't return a match. Is it just a flake of the upstream API-- for whatever reason, it wasn't able to parse a logo this time? If it hasn't been failing consistently (this PR passed, but should have failed under the same conditions), I think we can just leave it.

Also a note, the [assert.ok()(https://nodejs.org/api/assert.html#assert_assert_ok_value_message) signature is `assert.ok(isThisValueTrue, 'optional message to print if `isThisValueTrue` is false')`.

Reply:
I see I will leave as it is.